documents.go

```diff
@@ -78,16 +78,16 @@ type DocumentsCreateClient struct {
 	params documentsCreateParams
 }
 
-func newDocumentsCreateClient(sl *sling.Sling, params documentsCreateParams) *DocumentsCreateClient {
+func newDocumentsCreateClient(sl *sling.Sling, title string, collectionId CollectionID) *DocumentsCreateClient {
```

Comment:
collectionId -> id; the type shows the param is representing

Reply:
Fixed here https://github.com/ioki-mobility/go-outline/pull/40/commits/d87d1c4e9af40811acd4c67a2fe0ee9ad2b0aad4